engine/installation/linux/oracle.md

```diff
@@ -144,9 +178,24 @@ Docker from the repository.
     $ sudo yum -y install docker-engine-<VERSION_STRING>
```

Comment:
I would remove the `-y` option from this command-line example. It would suppress the GPG key validation check.